addons/l10n_uy/demo/res_currency_rate_demo.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+
+    <record id="rateUSD" model="res.currency.rate">
+        <field name="company_id" ref="l10n_uy.demo_company_uy"/>
+        <field name="rate">0.026</field>
+        <field name="currency_id" ref="base.USD"/>
+        <field eval="time.strftime('2023-03-02')" name="name"/>
```

Comment:
Is this date random? Shouldn't we do something like 01/01/2023 or some date like this, so it seems more consistent?